lib/diagnostic.rb

```diff
@@ -108,10 +108,17 @@
 # Use shorthand to make this one line long.
 # Write your response as a Ruby string.
 
-Diagnostic.person_hash = 'your answer here'
+Diagnostic.person_hash =
+class Person
+  def initialize(favorite_number, first_name)
+    @favorite_number = favorite_number
+    @first_name = first_name
+  end
+end
+first_person = new.Person(3, 'Dee Dee')
 
 # Set a default return value to your `person` hash that concatenates the value
 # of person[:first_name] and a message about them.
 # Write your response as a Ruby string.
 
-Diagnostic.hash_default = 'your answer here'
+Diagnostic.hash_default = p "#{first_person.first_name} is a very nice person."
```

Comment:
Close but remember, the dot notation in Ruby will call a method. Here's what we were looking for:

`'person.default = "#{person[:first_name]} is a very nice person."'`